package.json

```diff
@@ -14,11 +14,12 @@
     "migration": "sequelize db:migrate",
     "seed": "sequelize db:seed:all",
     "reverse": "sequelize db:migrate:undo:all",
+    "refreshdb": "sequelize db:migrate:undo:all && sequelize db:migrate",
     "coveralls-coverage": "coveralls < lcov.info",
     "coverage": "nyc npm test && npm run generate-lcov && npm run coveralls-coverage && npm run codeclimate-coverage",
     "codeclimate-coverage": "codeclimate-test-reporter < lcov.info",
     "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
-    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
+    "test": "cross-env NODE_ENV=test npm run heroku-postbuild && nyc _mocha --require @babel/register  src/test/**.test.js --timeout 20000 --exit"
```

Comment:
Is there a reason we are running `heroku-postbuild` when the test script runs?

Reply:
yeah, so it can undo migration, run migration and seed the test 
database

Reply:
Okay. That script is strictly for `heroku-postbuild`, we shouldn't use it here so we don't call it unnecessary if it is later updated with something specifically for Heroku. You can call the scripts to undo migration, run migration and seed test here or create a new script combining them together.

Reply:
ok have implemented it